File: csharp/exercises/Webshop.UnitTests/PasswordValidatorTests.cs
using NUnit.Framework;
using Webshop.Service.Common;

namespace Webshop.UnitTests;

[TestFixture]
public class PasswordValidatorTests
{
    [Test]
    public void MyFirstTest()
    {
        // Arrange
        var passwordValidator = new PasswordValidator();
        // Act
        var isValid = passwordValidator.IsValidPassword("");
        // Assert
        // TODO: Write your assertions here
    }
}

File: java/exercises/src/test/java/Webshop/UnitTests/PasswordValidatorTests.java
package Webshop.UnitTests;

import Webshop.Service.Common.PasswordValidator;
import org.junit.jupiter.api.Test;

public class PasswordValidatorTests {

    @Test
    public void myFirstTest() {
        // Arrange
        PasswordValidator passwordValidator = new PasswordValidator();

        // Act
        boolean isValid = passwordValidator.isValidPassword("");

        // Assert
        // TODO: Write your assertions here
    }
}

// mvn -Dtest=PasswordValidatorTests#myFirstTest test